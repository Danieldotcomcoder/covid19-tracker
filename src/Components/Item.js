import PropTypes from 'prop-types';
import img from '../assets/virus.png';
import rimg from '../assets/right.png';

const Item = ({ confirmed, name }) => (
  <div className="Home-item-content">
    <div className="item-image">
      <img src={rimg} alt="right-arrow" />
    </div>
    <div className="Home-item-top">
      <img src={img} alt="virus view" className="Home-item-image" />
    </div>
    <div className="Home-item-bottom">
      <h2 className="country-name">{name}</h2>
      <h5 className="country-cases">{confirmed}</h5>
    </div>
  </div>
);

Item.propTypes = {
  confirmed: PropTypes.number.isRequired,
  name: PropTypes.string.isRequired,
};

export default Item;
