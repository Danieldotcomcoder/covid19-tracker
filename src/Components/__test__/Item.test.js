import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../../Redux/configureStore';
import Item from '../Item';

it('Renders item component', () => {
  const itemcomponent = renderer.create(
    <Provider store={store}>
      <Item />
      ,
    </Provider>,
  ).toJSON();
  expect(itemcomponent).toMatchSnapshot();
});
