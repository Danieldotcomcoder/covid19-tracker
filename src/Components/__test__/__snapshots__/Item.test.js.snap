// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders item component 1`] = `
Array [
  <div
    className="Home-item-content"
  >
    <div
      className="item-image"
    >
      <img
        alt="right-arrow"
        src="right.png"
      />
    </div>
    <div
      className="Home-item-top"
    >
      <img
        alt="virus view"
        className="Home-item-image"
        src="virus.png"
      />
    </div>
    <div
      className="Home-item-bottom"
    >
      <h2
        className="country-name"
      />
      <h5
        className="country-cases"
      />
    </div>
  </div>,
  ",",
]
`;
